src/routes/controllers/productsControllers/deleteProductByID.js

```diff
@@ -0,0 +1,23 @@
+import ProductModel from '../../../models/product';
+
+const deleteProductByID = async (req, res) => {
+  try {
+    const productID = await ProductModel.findById(req.params.productID);
+
+    if (!productID) return res.status(404).json({ msg: 'Product not found' });
+
+    const userId = req.params.userId;
+
+    if (!userId) return res.status(400).json({ msg: 'Please enter a user id' });
+
+    if (productID.user.toString() !== userId)
+      return res.status(401).json({ msg: 'Not authorized' });
+
+
+    await ProductModel.findByIdAndDelete(productID);
```

Comment:
can you see if your can actually return the deleted product. 

see if `const product = await ProductModel.find...` so you can return the deleted product to frontend?

Reply:
It also gets deleted from the frontend.